solutions/deploy-cluster-proxy-addon/README.md

```diff
@@ -0,0 +1,42 @@
+# Deploy cluster-proxy addon
+ 
+## Prerequisite
+
+Set up the dev environment in your local machine following [setup dev environment](../setup-dev-environment).
+
+## Support provisioning LoadBalancer typed service for kind
```

Comment:
the latest release of cluster-proxy doesnt require LoadBalancer anymore. can we clarify that this step is optional?

Reply:
alright.😁